src/source-napatech.c

```diff
@@ -376,28 +450,26 @@ TmEcode NapatechDecode(ThreadVars *tv, Packet *p, void *data, PacketQueue *pq,
     SCReturnInt(TM_ECODE_OK);
 }
 
-TmEcode NapatechDecodeThreadInit(ThreadVars *tv, const void *initdata, void **data)
-{
+TmEcode NapatechDecodeThreadInit(ThreadVars *tv, const void *initdata, void **data) {
```

Comment:
please undo this as our style guide says to have the opening braces in functions on a new line: https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Coding_Style#Braces